.github/workflows/PublishIncrementalNuget.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish Incremental Nuget Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v4
+  
+      - name: Setup .NET
```

Comment:
Can we extract repeated things to somewhere? Would be nice to re-use same things in different workflows

Reply:
I have no idea. I created these a long time ago and I just keep copy-pasta'ing them everywhere because they work! :)

Reply:
can we do this for a later checkin? I want to work more in themes area and this delta renamed everything...